### DISCUSSION TITLE
Activity timestamps

### DISCUSSION THREAD

**Discussion by wesraynbirdtilbury** [philosowaffle/peloton-to-garmin#449] [Q&A] [answered]
Sorry if I missed this in the documentation, but i've noticed all my synced activities have the same timestamp `12:00:00`, rather than the actual timestamp of the activity (note the dates are fine, but the time is wrong).

Has anyone else experienced this? Is this a known feature, a bug and is there a fix?

**Comment by pracademic** [comment]
I am having a somewhat similar issue, all my workouts on Garmin are showing up as 1 hour before the workout on peloton

**Reply by wesraynbirdtilbury** [reply]
The timestamps seem to appear ~6 hours before the time of the activity?
Recently I've had an upload at 01:40AM (actual time was 07:39AM) and 09:55AM (actual time was 15:54AM).

@philosowaffle are you able to advise? Thanks

**Reply by wesraynbirdtilbury** [reply]
This could be to do with the `TZ` argument [here](https://github.com/philosowaffle/peloton-to-garmin/discussions/303) - I've just changed mine to `Europe/London` so I'll see if that has an impact

**Reply by wesraynbirdtilbury** [reply]
I can confirm this is just a timezone argument issue, you just need to update your `TZ` argument to the timezone that you completed the activity within. 

The 1 hour difference is because you've a 1 hour time difference with the `America/Chicago` timezone.

**Reply by pracademic** [reply]
thank you all!

**Comment by wesraynbirdtilbury** [comment, marked as answer]
Check the `TZ` (timezone) [argument](https://github.com/philosowaffle/peloton-to-garmin/discussions/303)
